libbeat/scripts/kibana_export.py

```diff
@@ -0,0 +1,47 @@
+from elasticsearch import Elasticsearch
+import argparse
+import os
+import json
+
+
+def dump_objects(es, output_directory, doc_type):
+    res = es.search(
+        index='.kibana',
```

Comment:
The index where the Kibana dashboards are saved needs to be configurable. Adding a new argument to the command line, that it's not required, would be enough.